hawkbit-ddi-resource/src/main/java/org/eclipse/hawkbit/ddi/rest/resource/DdiRootController.java

```diff
@@ -251,14 +253,28 @@ private static boolean checkModule(final String fileName, final SoftwareModule m
 
             final HandlingType handlingType = action.isForce() ? HandlingType.FORCED : HandlingType.ATTEMPT;
 
-            final DdiDeploymentBase base = new DdiDeploymentBase(Long.toString(action.getId()),
-                    new DdiDeployment(handlingType, handlingType, chunks));
+            DdiPreviousActionFeedback previousActionFeedback = null;
+            Optional<Action> actionWithDetails = controllerManagement.findActionWithDetails(actionId);
+
+            if (actionWithDetails.isPresent()) {
```

Comment:
It does make sense to prune out the server generated messages. Also I think the name "previousfeedback" is a bit confusing, rather "targethistory" seems more apt since we are only interested in what the target/controller reported.